plugins/module_utils/network/vyos/utils/utils.py

```diff
@@ -113,7 +113,7 @@ def get_lst_same_for_dicts(want, have, lst):
 
 
 def list_diff_have_only(want_list, have_list):
-    """
+    """ -
```

Comment:
Is this a typo?

Reply:
@danielmellado  yeah this is a typo, thank you.